crates/daphne/src/lib.rs

```diff
@@ -815,7 +820,7 @@ impl deepsize::DeepSizeOf for DapAggregationParam {
 
 impl Encode for DapAggregationParam {
     fn encode(&self, bytes: &mut Vec<u8>) -> Result<(), CodecError> {
-        let _ = bytes;
+        _ = bytes;
```

Comment:
nit: Revert this change if it's not related to this PR and the compiler doesn't require it.